smoketest/scripts/cli/test_container.py

```diff
@@ -96,6 +96,18 @@ def test_basic(self):
         tmp = cmd(f'sudo podman exec -it {cont_name} sysctl kernel.msgmax')
         self.assertEqual(tmp, 'kernel.msgmax = 4096')
 
+    def test_name_server(self):
+        cont_name = 'dns-test'
+        name_server = '192.168.0.1'
+        self.cli_set(base_path + ['name', cont_name, 'allow-host-networks'])
+        self.cli_set(base_path + ['name', cont_name, 'image', cont_image])
+        self.cli_set(base_path + ['name', cont_name, 'name-server', name_server])
+
+        self.cli_commit()
+
+        n = cmd_to_json(f'sudo podman inspect {cont_name}')
+        self.assertEqual(n['HostConfig']['Dns'][0], name_server)
+
```

Comment:
Just beeing super correct here, can you please also read /etc/resolv.conf from within the container to be 100% sure?

This file should have an example hos to acces fles insode a container.

Other then that tiny thing, LGTM

Reply:
added